tests/scripts/sync-files-to-vm.sh

```diff
@@ -4,8 +4,8 @@ set -eo pipefail
 
 source scripts/vars.env
 
-NGF_DIR=$(dirname "$PWD")
+NGF_DIR=$(dirname "$PWD")/
 
 gcloud compute config-ssh --ssh-config-file ngf-gcp.ssh >/dev/null
 
-rsync -ave 'ssh -F ngf-gcp.ssh' "${NGF_DIR}" username@"${RESOURCE_NAME}"."${GKE_CLUSTER_ZONE}"."${GKE_PROJECT}":~
+rsync -ave 'ssh -F ngf-gcp.ssh' "${NGF_DIR}" username@"${RESOURCE_NAME}"."${GKE_CLUSTER_ZONE}"."${GKE_PROJECT}":~/nginx-gateway-fabric
```

Comment:
Were there issues with this script?

Reply:
I think I might be the only one that had a problem, because my fork is still NKG so it wouldn't copy to the expected directory.
But this way it doesn't matter what you directory is called, it will always be copied to `nginx-gateway-fabric` on the VM.